### ISSUE TITLE
Conversion to Dalvik format failed with error 1 while compiling jitsi project

### ISSUE THREAD

**Issue by jagdishkk1989** [jitsi/jitsi-android#38]
I am able to fix all the missing errors and added missing library. But when I am going to compile then getting strange error.
<b>Conversion to Dalvik format failed with error 1 </b>

I think error is related to same class or package name. This error is getting due to <b>java_stubs.jar</b> because it contain the java.awt.* and javax.swing.* package that is already contain in <b>android.jar</b>.

I tried all trick and googled a lot but no solution. I am struggling since last 3-4 days. 
If anyone have idea. Please help me. I will be very thankful.

Thanks a lot in advance.

**Comment by cmeng-git** [comment]
The error may also be due to 64K limit exceeded on android. The jitsi-android source does not support multi-dex build. Note that java.awt and javax.swing are not supported on Android. The java_stub.jar is  designed to take care of these references as stubs.

I have started a new development on aTalk application – a fork from the jisti-android.  If you are interested, you may download the apk from the following site.
download_link=http://atalk.sytes.net/releases/atalk-android/aTalk-release.apk

**Comment by diegobill** [comment]
@cmeng-git , Do you have a github to atalk?

**Comment by cmeng-git** [comment]
No, aTalk source is still not available online. Actually I am in the mid of talking to some org to get atalk release.  Meantime I am still cleaning up the source.
